simulator/ns-3.35/src/point-to-point/model/switch-mmu.h

```diff
@@ -88,9 +88,9 @@ class SwitchMmu: public Object {
 	uint64_t xoffTotal;
 
 	// aggregate run time
-	uint64_t ingressPoolUsed;
+	uint64_t totalUsed;
 	uint64_t egressPoolUsed[2];
-	uint64_t totalSharedUsed;
```

Comment:
Got rid of this redundant variable `totalSharedUsed`. 
``totalUsed` can be used everywhere instead.